src/Std/Data/DHashMap/Internal/Defs.lean

```diff
@@ -352,6 +352,14 @@ where
     r := ⟨r.1.insert a b, fun _ h hm => h (r.2 _ h hm)⟩
   return r
 
+/-- Internal implementation detail of the hash map -/
+def insertList [BEq α] [Hashable α]
+    (m : Raw₀ α β) (l : List ((a : α) × β a)) : Raw₀ α β := Id.run do
+  let mut r := m
+  for ⟨a,b⟩ in l do
+    r:= r.insert a b
+  return r
```

Comment:
It's not really essentially, but for such a simple for loop I generally prefer using a `List.fold`.